sync2jira/upstream_issue.py

```diff
@@ -363,16 +382,13 @@ def github_issues(upstream, config):
 
         if not issue.get('storypoints', None):
             issue['storypoints'] = ''
-            graphqlurl = 'https://api.github.com/graphql'
-            orgname = "open-connectors"
-            reponame = "open-connectors"
+            orgname, reponame = upstream.split('/')
             issuenumber = issue['number']
```

Comment:
Since you intend for `split()` to produce exactly two results, consider specifying [the second argument](https://docs.python.org/3/library/stdtypes.html#str.split):
```python
            orgname, reponame = upstream.split('/', 1)
```

Reply:
Also, consider using [rsplit()](https://docs.python.org/3/library/stdtypes.html#str.rsplit) instead.

(It makes no difference unless and until `upstream` shows up with more than one `/` in it, which we probably hope never happens, but....)

Reply:
Done. 1afad545ee80433015e30fc217161ccda367d316